app/services/group_service.rb

```diff
@@ -9,11 +9,16 @@ def self.create(group:, actor: )
     return false unless group.valid?
 
     if group.is_parent?
+      if ab_testing_enabled?
+        group.segments['bx_choose_plan'] = [true, false].sample
```

Comment:
Things like this make me wish we'd made the push to extract our payment system into a plugin while we were doing the loomio_org plugin.